RELEASING.md

```diff
@@ -12,3 +14,15 @@ These are the steps to release Gradle profiler.
 * Update the [release draft](https://github.com/gradle/gradle-profiler/releases) prepared by [release-drafter](https://probot.github.io/apps/release-drafter/), assign the tag you created and publish the release
 * The [Publish to SDKman](https://builds.gradle.org/buildConfiguration/GradleProfiler_GradleProfilerPublishToSdkMan?branch=%3Cdefault%3E) should trigger automatically after the publishing job finishes. Though it may fail, since the artifacts are not yet on Maven Central. The build will retry the SDKMan publishing after half an hour. If it still doesn't work, you can trigger the build manually. 
 * All done!
+
+### Releasing Snapshot Versions
```

Comment:
❓ Should we talk about "alpha" versions instead?

Reply:
What about pre-release versions so it matches the GitHub terminology?

Reply:
Let's do that.